### ISSUE TITLE
TABERT: Pretraining for Joint Understanding of Textual and Tabular Data 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#29]
https://arxiv.org/abs/2005.08314
 A study of applying BERT to a question answering task with structured data. Using n-grams to select multiple candidate rows and then to merge the information of multiple rows by self-attention.  Unsupervised pre-training is done by masking the table  and filling in the holes like BERT.

![image](https://user-images.githubusercontent.com/24524018/89604168-a5239f00-d8a5-11ea-9220-40019e02a4d3.png)
